docs/source/en/quicktour.md

```diff
@@ -130,7 +130,7 @@ You can also use the pipeline locally. The only difference is you need to downlo
 Then load the saved weights into the pipeline:
 
 ```python
->>> pipeline = DiffusionPipeline.from_pretrained("./stable-diffusion-v1-5", use_safetensors=True)
+>>> pipeline = DiffusionPipeline.from_pretrained("./stable-diffusion-v1-5", local_files_only=True, use_safetensors=True)
```

Comment:
Sorry I don't fully understand why this is easier for the user? Loading the model works perfectly well without passing `local_files_only=True` no?

Reply:
Ah I must've misunderstood the purpose of `local_files_only`.  The docstring makes it sound like you should set it to `True` to avoid downloading the model from the Hub again and instead use the specified weights in a directory.

Reply:
Nono nothing is downloaded when a dir is passed :-) So not really necessary here. It's only necessary if you pass a repo_id but have no internet